Aptoide/aptoide-client-v8
# [Feature/ASV-1565] Deprecate share timeline

**What does this PR do?**

   Remove user/shareTimeline Requests.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Test share with your followers in the appview action button.
  Test recommends install button. It should not fail, but it should not make any request since the backend was removed to only return successfully.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1565](https://aptoide.atlassian.net/browse/ASV-1565)

**Code Review Checklist**
